tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSnsTests.cs

```diff
@@ -96,8 +98,8 @@ static string GetSnapshotSuffix(string packageVersion)
                     => packageVersion switch
                     {
                         // Lower versions than specified don't contain PublishBatch method
-                        null or "" => ".pre3.7.101.88",
-                        { } v when new Version(v) < new Version("3.7.101.88") => ".pre3.7.101.88",
```

Comment:
this was wrong, the value used in the sample app to determine if we do a batch request or not is 3.7.3, and it's the value used by default if there is no version specified.